packages/x-ui-collections-bundle/DOCUMENTATION.md

```diff
@@ -473,27 +473,122 @@ function Posts() {
       options: {
         // limit, skip, sort
       },
+    },
+    {
+      // This is an optional argument
+      apollo: {
+        // Pass here any additional Apollo query option you may need
+        // https://www.apollographql.com/docs/react/data/queries/#usequery-api
+        fetchPolicy: "network-only",
+      },
     }
   );
 
   // data will be Post[] directly
+
+  // Same concept applies to useQueryOne, useLazyQuery and useLazyQueryOne.
 }
 ```
 
 For working with single data relations please use `collection.useQueryOne()` having the exact same api, except `data` is going to be `Post` instead of `Post[]`.
 
-When you perform a mutation, you can now trigger the cache, by doing so:
+##### Refetching fields
+
+When you perform a mutation, the modified fields will be automatically inferred based on the ones in the mutation input :
+
+```ts
+collection.updateOne(post._id, {
+  title: "My New Title",
+  text: "My new Text",
+});
+```
+
+Is exactly the same as :
 
 ```ts
 collection.updateOne(
   post._id,
   {
     title: "My New Title",
+    text: "My new Text",
   },
-  // Refetch Body:
   {
-    // _id: 1 is auto-populated
-    title: 1,
+    refetchBody: {
+      _id: 1,
+      title: 1,
+      text: 1,
+    },
+  }
+);
+```
+
+Of course, you're free to specify your own refetch body instead, which will fully override the inferred one. Even better : let's say you actually want to refetch the updated field plus an additional one. For this case, we expose the internal function `toQueryBody`, such as :
+
+```ts
+import { toQueryBody } from "@bluelibs/x-ui-collections-bundle";
```

Comment:
I'm wondering to avoid this import, couldn't we create this method on the collection class?

Reply:
Hm, it would duplicate the fnction on each instance. And I thought that this function can come handy in other cases where we want to transform any object to a query body. What do you think about it ?